beacon-chain/rpc/prysm/v1alpha1/validator/proposer.go

```diff
@@ -143,15 +144,20 @@ func (vs *Server) GetBeaconBlock(ctx context.Context, req *ethpb.BlockRequest) (
 		vs.setBlsToExecData(sBlk, head)
 	}()
 
-	localPayload, blobsBundle, err := vs.getLocalPayloadAndBlobs(ctx, sBlk.Block(), head)
+	localPayload, blobsBundle, override, err := vs.getLocalPayloadAndBlobs(ctx, sBlk.Block(), head)
```

Comment:
Can you rename `override` to `overrideBuilder`? Currently from the variable name it's not clear what we are overriding.